Resources/public/js/views/subitem/ez-subitemgridview.js

```diff
@@ -292,6 +293,21 @@ YUI.add('ez-subitemgridview', function (Y) {
         },
 
         /**
+         * Return an array containing the subitems that we want to append
+         * in the grid view.
+         *
+         * @method _getSubItemsToAppend
+         * @protected
+         * @param {EventFacade} e
+         */
+        _getSubItemsToAppend: function (e) {
+            var subitemNewCount = e.newVal ? e.newVal.length : 0,
```

Comment:
this condition, is not needed (and thus the 0 case is not covered), the `subitems` array can only grow, there's no way to have a new value to something else than a filled array.